src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
-    Microsoft ResX Schema 
-    
+  <!--
+    Microsoft ResX Schema
+
```

Comment:
@brendanzagaeski and I usually edit the `.resx` file in VS Windows and I guess it's choosing different whitespace settings. Are we going to have an IDE whitespace fight going forward? 😜 

Should we avoid whitespace changes in `Resources.resx` and `Resource.designer.cs`?

Normal C# files should be fine if it is using the `.editorconfig` file to format.

Reply:
I'm just editing this in VScode so it should be using the .editorconfig settings...

Reply:
Unfortunately, yes, the Visual Studio `.resx` resource editor recreates this comment with trailing spaces every time it saves the file. And unfortunately, as far as I can find, `.editorconfig` does not provide settings to trim trailing whitespace within comments in Visual Studio. The current `.editorconfig` settings do trim trailing whitespace outside of comments in both C# and XML files, but not within comments. Apparently Visual Studio Code works differently and does trim trailing whitespace within comments.

Maybe the simplest compromise for now would be to set `trim_trailing_whitespace = false` for `.resx` files in the `.editorconfig` file. That would leave the responsibility on contributors to trim the whitespace by hand as appropriate for any changed lines and not to change trailing whitespace for any other lines. I have submitted that change in <https://github.com/xamarin/xamarin-android/pull/4954>.

For this PR, I have pushed a commit to the PR branch that resets the `Resource.Designer.cs` and `Resources.resx` files to match what Visual Studio generates. Feel free to squash that into the other commit as this PR moves to completion.